modules/server-node/prisma-sqlite/schema.prisma

```diff
@@ -195,6 +214,18 @@ model OnchainTransaction {
   gasLimit        String
   gasPrice        String
 
+  createdAt DateTime @default(now())
+
+  onchainTransaction   OnchainTransaction? @relation(fields: [onchainTransactionId], references: [id])
+  onchainTransactionId String
+
+  receipt OnchainTransactionReceipt? @relation(fields: [transactionHash], references: [transactionHash])
+  @@map(name: "onchain_transaction_attempt")
+}
+
+model OnchainTransactionReceipt {
+  // Response fields
+  transactionHash   String   @id
```

Comment:
So, one could make the argument that this hash should be omitted here as it is redundant. The receipt should apply to the *last* tx attempt in the tx attempt list column of the parent `OnchainTransaction`.

But gut feeling says we should keep this here for simplicity's sake